roles/docker-registry/handlers/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+- name: start docker registry
+  service: name=docker-registry state=started
+
+- name: stop docker registry
+  service: name=docker-registry state=stopped
+
+- name: restart docker registry
+  service: name=docker-registry state=restarted
```

Comment:
stopped and restarted handlers don't seem to be used, however restarted handler should maybe be used when configuration changes? Stopped could just go away.